test/e2e/helpers/smtpEmails.ts

```diff
@@ -20,6 +20,10 @@ const EmailResponseSchema = z.object({
   results: z.array(EmailItemSchema),
 })
 
+// removing smtp4dev url since only one platform
+const smtp4devUrl = process.env.VERITABLE_SMTP_ADDRESS || 'http://localhost:5001'
```

Comment:
so this is now both in the e2e test file and here? could we maybe just have it in one of the files and pass it from there just so it doesn't have to be in all files.

Reply:
yea, so it should be here, since this is the same URL across all nodes, did not want to do too much refactor, but should have updated, so will do now.

Reply:
Ok, so this has been updated along with `cleanup` method that takes array of Urls and woudl call `/reset` endpoint